polkadot/node/network/statement-distribution/src/v2/requests.rs

```diff
@@ -553,7 +553,6 @@ impl UnhandledResponse {
 		let UnhandledResponse {
 			response: TaggedResponse { identifier, requested_peer, props, response },
 		} = self;
-
```

Comment:
why remove newline ?

Reply:
Touched those while I was debugging, I shouldn't had included them in the commit, fixed now.